plugins/catalog/src/components/FilteredTableLayout/FilterContainer.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Box, Drawer, Grid, useMediaQuery } from '@material-ui/core';
+import { useEntityListProvider } from '@backstage/plugin-catalog-react';
+import { BackstageTheme } from '@backstage/theme';
+import React from 'react';
+
+export const FilterContainer = ({ children }: React.PropsWithChildren<{}>) => {
+  const isMidSizeScreen = useMediaQuery<BackstageTheme>(theme =>
+    theme.breakpoints.down('md'),
+  );
+  const { showFiltersDrawer, toggleFiltersDrawer } = useEntityListProvider();
+
+  return isMidSizeScreen ? (
+    <Drawer
+      data-testid="entity-filters-drawer"
+      open={showFiltersDrawer}
+      onClose={() => {
+        toggleFiltersDrawer(false);
+      }}
+      PaperProps={{
+        style: { width: '300px' },
```

Comment:
Why the magic number? What's its intent?

Reply:
Initially I set it to `300px`, because thats a magic number I saw in the `Select.tsx` component for the `max-width` of the select.

```ts
const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    formControl: {
      margin: `${theme.spacing(1)} 0px`,
      maxWidth: 300,
    }, 
  [...]
);
```

I agree with you that enforcing a static width here is not a good solution. My take now is, that I removed it, as the initial width is actually already good & if smth. added to the filters will take more space it will than scale:
 
<img width="793" alt="Screenshot 2021-07-16 at 14 17 08" src="https://user-images.githubusercontent.com/8904624/125946046-7c7f5537-7cb8-4a5f-be81-8fb68ddf1974.png">